test/integration/EndToEnd.t.sol

```diff
@@ -66,17 +66,17 @@ contract TestEndToEnd is Test {
         deploy.removeFullDeployerAccess(address(deploy));
     }
 
-    function _getDeploys(bool sameChain) public view returns (PoolsDeployer cp, VaultsDeployer cv) {
-        cp = deployA;
+    function _getDeploys(bool sameChain) public view returns (HubDeployer ch, VaultsDeployer cv) {
+        ch = deployA;
```

Comment:
Not a huge fan of this acronym, think it creates more confusion than is helpful, but let's merge as is and we can consider renaming this later

Reply:
I agree. We should find a new one or just `h` in this case to avoid `cp`.